app/src/main/java/org/simple/clinic/editpatient/PatientEditScreen.kt

```diff
@@ -109,7 +109,6 @@ class PatientEditScreen(context: Context, attributeSet: AttributeSet) : Relative
     }
```

Comment:
What do you think about renaming this commit to just `Show confirm discard dialog on back press`?

Reply:
Sounds good. I'll make this.